libs/execution_context/ebpf_link.c

```diff
@@ -485,6 +485,8 @@ _ebpf_link_instance_invoke_batch_begin(
         goto Done;
     }
 
+    memset(execution_context_state, 0, sizeof(ebpf_execution_context_state_t));
```

Comment:
Is there a runtime cost to initializing the entire struct? I was wondering if we should only zero-initialize fields that won't be initialized later in this routine?

Reply:
On my machine, for example, we seemed to run into problem with the tail_call fields. Maybe that's all that needs to be zero'd?

Reply:
I think merge it now, then measure to see if there is a significant difference in performance.